src/app/tabs/runTab/model/dropdownlogic.js

```diff
@@ -106,25 +107,25 @@ class DropdownLogic {
 
   fromWei (value, doTypeConversion, unit) {
     if (doTypeConversion) {
-      return executionContext.web3().fromWei(typeConversion.toInt(value), unit || 'ether')
+      return Web3.utils.fromWei(typeConversion.toInt(value), unit || 'ether')
```

Comment:
Why not using ethers.js instead as we're moving out of Web3 to ethers.js ?

Reply:
I just wanted to get rid of the using executionContext here avoiding breaking changes as maximum as possible. I agree later we can refactor this to either use web3-utils directly or something else like etherjs, it's quite possible this whole method might disapear though so optimizing now could be premature